vcd/validate_funcs.go

```diff
@@ -112,3 +112,24 @@ func validateMultipleOf4() schema.SchemaValidateFunc {
 		return
 	}
 }
+
+// IntLeaseSeconds validates amount of seconds for lease
+// A value of 0 is accepted, as it means "never expires"
+// Regular values must be > 3600
+func IntLeaseSeconds() schema.SchemaValidateFunc {
```

Comment:
currently our validation functions starts with checkSomething, validateSomething. Maybe there is better name for this function.

Reply:
Thanks. Renaming. I used the current name modeling on `validation.IntAtLeast` and similar. But consistency within our library is important.